source/macos/AdaptiveCards/AdaptiveCardsTests/Renderers/InputDateRendererTests.swift

```diff
@@ -15,7 +15,7 @@ class InputDateRendererTest: XCTestCase {
     }
 
     func testRendererSetsValue() {
-        let val: String = "23/11/2000"
+        let val: String = "2000-10-12"
         inputDate = .make(value: val)
```

Comment:
Why are these changed?
Was the change causing any regression?

Reply:
Yeah, the formatter expects the date in yyyy-mm-dd format
dd/mm/yyyy was the old format which should not be supported. It was fixed in the code but never changed here as the tests were still passing I guess